README.md

```diff
@@ -1 +1,31 @@
-# OOP-school-library
\ No newline at end of file
+# OOP school library
+This is an "OOP school library" app, that records what books are in the library and who borrows them.
+
+# Built With
+  Ruby
+To get a local copy run the following steps:
+Copy this [link](https://github.com/hamid-murambiwa/enumerable.git)
+Open your terminal or command line
+Run `git clone https://github.com/hamid-murambiwa/enumerable`
+Open the folder with your code editor
+Create a branch to work on
+Or open the list.rb file in the dist directory to start coding
+
+# Author
+
+👤 Hamid Murambiwa
+
+-GitHub: [@githubhandle](https://github.com/hamid-murambiwa/)
+-Twitter: [@twitterhandle](https://twitter.com/Hamid87789454/)
+-LinkedIn: [LinkedIn](https://linkedin.com/in/hamid-murambiwa/)
+
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Acknowledgements
+Show your support
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] As it is a requirement for this project, please kindly update your `readme file` following this [template](https://github.com/microverseinc/readme-template), as will make your `readme file` more informative and professional.